wagtailio/utils/choices.py

```diff
@@ -3,40 +3,52 @@
 
 class SVGIcon(models.TextChoices):
     ARROW = "arrow", "Arrow"
-    BI_ARROW = "bi-arrow", "Bi-arrow"
+    ARROW_ALT = "arrow-alt", "Arrow alt"
```

Comment:
let's move these to the core module. My goal is for us to ultimately remove utils

Reply:
done in 9c0c2fc3d9349a3a3d3e4f3f81fb691987d1699d